ci-operator/step-registry/ipi/deprovision/proxy/ipi-deprovision-proxy-ref.yaml

```diff
@@ -0,0 +1,13 @@
+ref:
+  as: ipi-deprovision-proxy
+  from_image:
+    namespace: ocp
+    name: "4.5"
+    tag: upi-installer
```

Comment:
Hmm, we must be pinning this because of the aws client? Looks like yes: openshift/installer#3292. Can we call that out in the commit message? And maybe drop the 4.5-pinning for gather-proxy-ref.yaml now that it no longer uses aws?